common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1214 @@
+                  value: |
+                    2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
+                  value: |
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_CREATE_CLUSTER_ROLES
+                  value: "TRUE"
+                - name: STRIMZI_LOG_LEVEL
+                  value: INFO
+                - name: STRIMZI_CUSTOM_KAFKA_BRIDGE_SERVICE_LABELS
+                  value: |
+                    discovery.3scale.net=true
+                - name: STRIMZI_CUSTOM_KAFKA_BRIDGE_SERVICE_ANNOTATIONS
+                  value: |
+                    discovery.3scale.net/scheme=http
+                    discovery.3scale.net/port=8080
+                    discovery.3scale.net/path=/
+                    discovery.3scale.net/description-path=/openapi
+                livenessProbe:
+                  httpGet:
+                    path: /healthy
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                readinessProbe:
+                  httpGet:
+                    path: /ready
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                resources:
+                  limits:
+                    cpu: 1000m
+                    memory: 256Mi
+                  requests:
+                    cpu: 200m
+                    memory: 256Mi
+          strategy:
+            type: Recreate
+  customresourcedefinitions:
+    owned:
+    - name: kafkas.kafka.strimzi.io
+      version: v1beta1
+      kind: Kafka
+      displayName: Kafka
+      description: Represents a Kafka cluster
+      resources:
+      - kind: Route
+        name: ''
+        version: route.openshift.io/v1
+      - kind: Service
+        name: ''
+        version: v1
+      - kind: StatefulSet
+        name: ''
+        version: v1
+      - kind: Deployment
+        name: ''
+        version: v1
+      - kind: ReplicaSet
+        name: ''
+        version: v1
+      - kind: Pod
+        name: ''
+        version: v1
+      - kind: Secret
+        name: ''
+        version: v1
+      specDescriptors:
```

Comment:
I think we had some updates for this section in Strimzi. Did you added those here? (the review of these sucks since it is adding new files :-()

Reply:
yes I added the ones for KafkaConnector and KafkaMirrorMaker2